### ISSUE TITLE
Allow to set time for `Delayed descriptions for characters on cursor movement`

### ISSUE THREAD

**Issue by munawarb** [nvaccess/nvda#14470]
### Is your feature request related to a problem? Please describe.

The time between NVDA announcing a character and it's description should be customizable.

### Describe the solution you'd like

An edit box to set the amount of milliseconds to wait before reading the description. I feel that 1 second is too long in some cases and this is a personal preference that should be changeable.

**Comment by seanbudd** [comment]
I believe this is a duplicate of https://github.com/nvaccess/nvda/issues/13967, feel free to add discussion there if you think anything is missed
